### PR TITLE
Updated course view

### PR DESCRIPTION
**Pull request by nozymko**
Now, the newest assignments should be on the top, and assignments with no available run should be greyed out.

### CODE REVIEW

File: src/routes_student.py [hunk 1]
```diff
@@ -25,6 +25,19 @@ def student_get_course(netid, cid):
 
 			course = db.get_course(cid)
 			assignments = db.get_assignments_for_course(cid)
+			now = util.now_timestamp()
+
+			for assignment in assignments:
+
+				num_extension_runs = get_active_extensions(cid, assignment["assignment_id"], netid, now)
+				num_available_runs = get_available_runs(cid, assignment["assignment_id"], netid, now)
+				total_runs = num_extension_runs[1] + num_available_runs
```
Comment (ayushr2) [inline src/routes_student.py]: I think this should be renamed as `total_available_runs`.

File: src/routes_student.py [hunk 2]
```diff
@@ -25,6 +25,19 @@ def student_get_course(netid, cid):
 
 			course = db.get_course(cid)
 			assignments = db.get_assignments_for_course(cid)
+			now = util.now_timestamp()
+
+			for assignment in assignments:
+
+				num_extension_runs = get_active_extensions(cid, assignment["assignment_id"], netid, now)
```
[the remark this answers is not in the document]
Comment (nozymko) [reply src/routes_student.py]: Done! I ended up copying how some code uses get_active_extensions later in the file

File: src/routes_student.py (hunk 1 above)
Comment (nozymko) [reply src/routes_student.py]: Done!

**Comment by ayushr2** [comment]
It is good practice to reference the issue you are solving in the PR description. If you write "Resolves #79", it will automatically close the issue when the PR is merged.

For UI related PRs, it is also useful to attach images (the same ones you had on Slack would work).

**Pull request merged**